phoenix-core/src/main/java/org/apache/phoenix/schema/types/PDouble.java

```diff
@@ -17,15 +17,16 @@
  */
 package org.apache.phoenix.schema.types;
 
-import com.google.common.base.Preconditions;
-import com.google.common.primitives.Doubles;
```

Comment:
Ah, ok. My bad. Must have been the someone else before you. Thanks for correcting. I'll get this committed. Great work!